opennms-config-jaxb/src/main/java/org/opennms/netmgt/config/datacollection/Group.java

```diff
@@ -271,6 +283,13 @@ public boolean equals(final Object obj) {
         } else if (!m_mibObjects.equals(other.m_mibObjects)) {
             return false;
         }
+        if (m_mibObjProperties == null) {
+            if (other.m_mibObjProperties != null) {
+                return false;
+            }
+        } else if (!m_mibObjProperties.equals(other.m_mibObjProperties)) {
```

Comment:
MibObjProperty does not implement hashCode() and equals(), so this may not work as expected.